src/docs/product/integrations/integration-platform/ui-components/issue-link.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Issue Link
+sidebar_order: 1
+description: "Learn about the issue link UI component."
+---
+
+With an issue linking UI component, you can connect Sentry issues with a task in your project management system. This functionality provides a way to use any project management tool you use or develop.
+
+The issue link component displays with the text "Link &lt;Service&gt; Issue" in the **Issue Details** sidebar:
+
+![Issue linking sidebar showing Shortcut icon.](issue-sidebar-icons.png)
+
+Clicking the link opens a modal allowing the user to create or link the issue to a task in the external service:
+![Sentry modal that creates a Shortcut Story that's linked to a Sentry issue.](../../issue-tracking/shortcut/shortcut_create_story_with_issue.png)
+
+## Issue Link Schema
```

Comment:
To be consistent with the other UI Component pages, I think think this header should be changed to just `Schema` (or you can change the others to this pattern)